valgrind.supp

```diff
@@ -57,3 +57,21 @@
    fun:_ULx86_64_step
    ...
 }
+
+{
+   tcmalloc trips valgrind on ubuntu 14
+   Memcheck:Param
+   msync(start)
+   fun:__msync_nocancel
+   obj:/usr/lib/x86_64-linux-gnu/libunwind.so.8.0.1
+   obj:/usr/lib/x86_64-linux-gnu/libunwind.so.8.0.1
+   obj:/usr/lib/x86_64-linux-gnu/libunwind.so.8.0.1
+   obj:/usr/lib/x86_64-linux-gnu/libunwind.so.8.0.1
+   fun:_ULx86_64_step
+   fun:_Z13GetStackTracePPvii
+   fun:_ZN8tcmalloc8PageHeap8GrowHeapEm
+   fun:_ZN8tcmalloc8PageHeap3NewEm
+   fun:_ZN8tcmalloc15CentralFreeList8PopulateEv
+   fun:_ZN8tcmalloc15CentralFreeList18FetchFromSpansSafeEv
+   fun:_ZN8tcmalloc15CentralFreeList11RemoveRangeEPPvS2_i
+}
```

Comment:
I think that would be because the object path changed: `/usr/lib/libunwind.so.7.0.0` vs `/usr/lib/x86_64-linux-gnu/libunwind.so.8.0.1`. I believe if you use the 8.0.1 path, the suppression should work.